styles.css

```diff
@@ -0,0 +1,2153 @@
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .css {
+    width: 21px;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .ruby {
+    width: 100%;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .js {
+    width: 65px;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .pry-text {
+    width: 88%;
+    height: 76px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 13px;
+    line-height: 24px;
+    color: #344563;
+    margin-top: 0%;
+  }
+
+  .see-project {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 0;
+    position: relative;
+    top: 0%;
+    width: 130px;
+    height: 48px;
+    left: 0.8%;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    background-color: #fff;
+    cursor: pointer;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+  }
+
+  .see-project-l {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 0;
+    position: absolute;
+    width: 12%;
+    height: 48px;
+    left: 12.8%;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    background-color: #fff;
+    cursor: pointer;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+  }
+
+  .snap2 {
+    display: none;
+  }
+
+  #desk2 {
+    display: block;
+    width: 50%;
+    height: 112%;
+    justify-content: center;
+    border-radius: 8px;
+    margin-bottom: -37px;
+    order: 1;
+  }
+
+  #desk3 {
+    display: block;
+    width: 50%;
+    height: 112%;
+    justify-content: center;
+    border-radius: 8px;
+    margin-bottom: -37px;
+  }
+
+  #desk4 {
+    display: block;
+    width: 50%;
+    height: 112%;
+    justify-content: center;
+    border-radius: 8px;
+    margin-bottom: -37px;
+    order: 1;
+  }
+
+  .snap3 {
+    display: none;
+  }
+
+  .snap4 {
+    display: none;
+  }
+
+  .top {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 0%;
+    width: 30%;
+    height: 409px;
+    position: absolute;
+    left: 142px;
+  }
```

Comment:
- [ ] please ensure that the space between all elements is respected as shown in the Figma Design 👌, To easily achieve this I suggest just minimizing the margin values to those elements.

### Your Design
![image](https://user-images.githubusercontent.com/68030297/221351838-c1306a6c-fdd4-4eb9-944c-cbe37fd88464.png)

### Figa Design 
![image](https://user-images.githubusercontent.com/68030297/221351858-67d55203-d1d2-4afa-9c6d-72b1d99fc3bd.png)